src/xcode/ENA/ENA/Source/Extensions/ENTemporaryExposureKey+Processing.swift

```diff
@@ -5,6 +5,7 @@
 import Foundation
 import ExposureNotification
 
+// [KGA] Hier wäre Ziel, processedforSubmission zu nutzen für meine Keys. Wie alt ist der Checkin. Dann entsprechend transmissionRiskLevel infektion zu weisen
```

Comment:
Can we change the comment to English?

Reply:
Oh we missed that one. All `[KGA]` comments will be removed :)